### ISSUE TITLE
Remove unstable key-value attributes

### ISSUE THREAD

**Issue by emmaling27** [vinted/elasticsearch-dsl-rs#42]
`elasticsearch_dsl` does not compile on stable Rust, which is a blocker for my company's use of this crate. It looks like the only unstable feature is [this one](https://github.com/rust-lang/rust/issues/78835), and it would only change one line of code: `#![doc = include_str!("../README.md")]`. Thoughts on removing this?

**Comment by buinauskas** [comment]
@emmaling27 that sounds quite weird, I'm using stable 1.55.0 and it compiles great, CI builds are also using stable rust version.

If only the `#![doc = include_str!("../README.md")]` is the cause - I'm happy to hardcode docs in the module, it's not a big deal. 

Ability to include files to directly include file contents as readme was added in 1.54.0, from [release notes](https://blog.rust-lang.org/2021/07/29/Rust-1.54.0.html):
> Rust 1.54 supports invoking function-like macros inside attributes. Function-like macros can be either `macro_rules!` based or procedural macros which are invoked like `macro!(...)`. One notable use case for this is including documentation from other files into Rust doc comments. For example, if your project's README represents a good documentation comment, you can use `include_str!` to directly incorporate the contents. Previously, various workarounds allowed similar functionality, but from 1.54 this is much more ergonomic.
> 
> ```
> #![doc = include_str!("README.md")]
> ```

The issue most likely is you're using an older version, https://github.com/vinted/elasticsearch-dsl-rs/pull/45 this fixes it. I'll release a new version today so you could use it in your project ;)

**Comment by emmaling27** [comment]
You're 100% right, I forgot we haven't updated rust for a while. Thanks for making the change anyway! (Updating will be a large project).

**Comment by buinauskas** [comment]
@emmaling27 cheers mate, 0.1.3 is available in crates.io already, so this should unblock your work ;)

**Comment by buinauskas** [comment]
@emmaling27 I believe this is fixed and I'm closing it, create another issue if this reoccurs. Thanks for the report!
